datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/VersionController.java

```diff
@@ -75,10 +95,18 @@ public void readVersionedDocument(@RequestHeader String user,
      */
     @SneakyThrows
     @PutMapping(value = "/versioned/{path:.*}", consumes = APPLICATION_OCTET_STREAM_VALUE)
+    @ApiOperation("Write latest document to user's private space")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Document was successfully written"),
+            @ApiResponse(code = 403, message = "Access denied")
```

Comment:
403 has default message, not necessary to add it

Reply:
removed